packages/terra-file-path/tests/jest/FilePath.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+/* eslint-disable-next-line import/no-extraneous-dependencies */
+import { shallowWithIntl } from 'terra-enzyme-intl';
+import FilePath from '../../src/FilePath';
+
+beforeAll(() => {
+  jest.spyOn(console, 'error').mockImplementation();
+  jest.spyOn(console, 'warn').mockImplementation();
+});
+
+afterEach(() => {
+  console.error.mockClear(); // eslint-disable-line no-console
+  console.warn.mockClear(); // eslint-disable-line no-console
+});
```

Comment:
Since, this is enabled in [jest-config-terra](https://github.com/cerner/terra-toolkit/blob/440ead889dfd75b31a3535e67d76179b0836dd71/packages/jest-config-terra/jest.config.js#L6), we no longer need to explicitly do a mockClear according to the docs:

https://jestjs.io/docs/mock-function-api#mockfnmockclear